src/FakeItEasy/Configuration/PropertySetterConfiguration.cs

```diff
@@ -101,10 +78,8 @@ public IPropertySetterConfiguration WhenArgumentsMatch(Func<ArgumentCollection,
         }
 
         private static IPropertySetterConfiguration AsPropertySetterConfiguration(
-            IVoidConfiguration voidArgumentValidationConfiguration)
-        {
-            return new ProperySetterAdapter(voidArgumentValidationConfiguration);
-        }
+            IVoidConfiguration voidArgumentValidationConfiguration) =>
+            new ProperySetterAdapter(voidArgumentValidationConfiguration);

```

Comment:
I don't mind reverting back to a block, but before I do, what's the opinion on this style?

```C#
private static IPropertySetterConfiguration AsPropertySetterConfiguration(
    IVoidConfiguration voidArgumentValidationConfiguration) =>
        new ProperySetterAdapter(voidArgumentValidationConfiguration);
```

I guess it also looks a little weird.